geometry_msgs/Point minPoint
geometry_msgs/Point size
std_msgs/Int8 resolution


